src/app/src/util/util.ga.js

```diff
@@ -0,0 +1,122 @@
+import env from './env';
+
+const REACT_APP_GOOGLE_ANALYTICS_KEY = 'REACT_APP_GOOGLE_ANALYTICS_KEY';
+
+export const GA_TRACKING = 'GA_TRACKING';
+export const HAS_ACCEPTED_GA_TRACKING = 'HAS_ACCEPTED_GA_TRACKING';
+export const HAS_REJECTED_GA_TRACKING = 'HAS_REJECTED_GA_TRACKING';
+
+
+export const maybeGetReactAppGoogleAnalyticsKey = () => {
+    const gaKey = env(REACT_APP_GOOGLE_ANALYTICS_KEY);
+    const environment = env('ENVIRONMENT');
+
+    if (!environment || environment !== 'development') {
```

Comment:
This is part of the test commit that will be dropped before merging. It's supposed to be `if (environment === development) return null;`.

Reply:
Yeah, sorry for not explaining well. I'll drop c68fabfa56b3e2a55addc9443e9915eb461ad863 before merging this, but it is essentially just to be able to test this in local development. For production and staging it will be set to `=== 'development'` -> `return null`